tests/cases/elements/popupwindow_close_policy.slint

```diff
@@ -259,6 +259,21 @@ assert_eq!(instance.get_click_count(), 2);
 assert_eq!(instance.get_popup_clicked(), 2003);
 
 
+// --------- Close outside click popup by esc
+instance.set_popup_selector(3);
+instance.set_click_count(0);
+slint_testing::send_mouse_click(&instance, 5., 5.);
+assert_eq!(instance.get_click_count(), 1);
+
+// click inside
+slint_testing::send_mouse_click(&instance, 15., 15.);
+assert_eq!(instance.get_popup_clicked(), 2005);
+
+// close by esc
```

Comment:
Please also add a esc press around line 216 (`Click outside, nothing happens`) to make sure that clicking on esc does not close that popup.


And maybe also extending popupwindow_nested.slint to test that esc closes one and only one popup.